.github/workflows/dashboard.yaml

```
@@ -0,0 +1,39 @@
+name: Deploy to GitHub Pages
+
+on:
+  push:
+    branches:
+      - main
+      - dashboard-setup
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Set up NodeJS
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16
+
+      - name: Install pnpm
+        run: npm install -g pnpm
+
+      - name: Navigate to Nuxt project folder & Install dependencies
+        run: |
+          cd _dashboard/DotNetPerfMonitor/
```

I think you can use `working-directory` instead:
```suggestion
          working-directory: _dashboard/DotNetPerfMonitor/
```